### ISSUE TITLE
[ossia-max] Feature request: Package version

### ISSUE THREAD

**Issue by maybites** [ossia/libossia#806]
Hi

currently the version tag inside the package is used to store the github commit:

https://github.com/ossia/libossia/blob/514c7f5bf37a6376779dd0b42fa6c2802dc93ab9/src/ossia-max/package-info.json.in#L31

but when one is looking at the package, this is not useful for the user (or me for instance: I parse this info to do a version check for dependency packages) to see which version is actually installed.

Would it be possible to use a different key to store the commit-hash instead, and store the actual release nummber inside it?

**Comment by jcelerier** [comment]
makes sense yes. can try to turn it into a git tag if it's a commit that comes from a tag
